packages/web/src/utils/LocalStorage.ts

```
@@ -0,0 +1,37 @@
+  static set(key: LocalStorageType, value: string) {
+    localStorage.setItem(key, value);
+  }
+
+  static setTrue(key: LocalStorageType) {
+    this.set(key, LOCAL_STORAGE_VALUE_MAP.true);
+  }
+
+  static check(key: LocalStorageType) {
+    return this.get(key) === LOCAL_STORAGE_VALUE_MAP.true;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)